NuGet.config

```diff
@@ -6,8 +6,8 @@
     <add key="AI myget" value="https://www.myget.org/F/applicationinsights/api/v3/index.json" />
     <add key="dotnet.myget.org dotnet-cli" value="https://dotnet.myget.org/F/dotnet-cli/" />
     <add key="coreclr-xunit" value="https://www.myget.org/F/coreclr-xunit/api/v2" />
-    <add key="AspNetCIDev" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
     <add key="nugetbuild" value="https://www.myget.org/F/nugetbuild/api/v3/index.json" />
+    <add key="Aspnet Release" value ="https://myget.org/f/aspnetrelease" />
```

Comment:
I was little confused between vnext, dev and release branches. Thank you for the information. I removed it, and push the changes in the next commit.